packages/api-explorer/src/index.jsx

```diff
@@ -192,38 +190,44 @@ class ApiExplorer extends React.Component {
         >
           {docs.map((doc, index) => (
             <VariablesContext.Provider key={index} value={this.props.variables}>
-              <OauthContext.Provider value={this.props.oauth}>
-                <GlossaryTermsContext.Provider value={this.props.glossaryTerms}>
-                  <BaseUrlContext.Provider value={this.props.baseUrl.replace(/\/$/, '')}>
-                    <SelectedAppContext.Provider value={this.state.selectedApp}>
-                      <Doc
-                        key={doc._id}
-                        appearance={this.props.appearance}
-                        auth={this.state.auth}
-                        baseUrl={this.props.baseUrl.replace(/\/$/, '')}
-                        doc={doc}
-                        flags={this.props.flags}
-                        group={this.state.group}
-                        groups={this.groups}
-                        language={this.state.language}
-                        lazy={this.isLazy(index)}
-                        Logs={this.props.Logs}
-                        maskErrorMessages={this.props.maskErrorMessages}
-                        oas={this.getOas(doc)}
-                        oauth={this.props.oauth}
-                        onAuthChange={this.onAuthChange}
-                        onAuthGroupChange={this.onAuthGroupChange}
-                        onError={this.props.onError}
-                        setLanguage={this.setLanguage}
-                        suggestedEdits={this.props.suggestedEdits}
-                        tryItMetrics={this.props.tryItMetrics}
-                        useNewMarkdownEngine={this.props.useNewMarkdownEngine}
-                        user={this.props.variables.user}
-                      />
-                    </SelectedAppContext.Provider>
-                  </BaseUrlContext.Provider>
-                </GlossaryTermsContext.Provider>
-              </OauthContext.Provider>
+              <NewVariablesContext.Provider key={index} value={this.props.variables}>
+                <OauthContext.Provider value={this.props.oauth}>
+                  <GlossaryTermsContext.Provider value={this.props.glossaryTerms}>
+                    <NewGlossaryTermsContext.Provider value={this.props.glossaryTerms}>
+                      <BaseUrlContext.Provider value={this.props.baseUrl.replace(/\/$/, '')}>
+                        <NewBaseUrlContext.Provider value={this.props.baseUrl.replace(/\/$/, '')}>
+                          <SelectedAppContext.Provider value={this.state.selectedApp}>
```

Comment:
@erunion per some of my comments on [this issue](https://github.com/readmeio/api-explorer/issues/620), I've double-wrapped the explorer's doc contents in _both_ the old and new context providers. This is super verbose/redundant solution, but an interim one. I just haven't had a chance to go through and  switch out all the references.